frontend/Dockerfile

```diff
@@ -22,12 +22,20 @@ COPY yarn.lock .
 RUN yarn --frozen-lockfile
 
 FROM base AS production
-COPY --from=build /build/.next/ .next/
-COPY --from=build /deps/node_modules/ node_modules/
-COPY package.json .
-COPY next.config.js .
-COPY public/ public/
-RUN mkdir src/
-COPY src/server.js src/server.js
+
 ENV NODE_ENV=production
-CMD yarn start
+# Make files are readable execuatable bu the `node` user.
+ARG PERMISSION=0555
```

Comment:
```suggestion
# allow the node user to read files and nothing else
ARG PERMISSION=004
```

Reply:
I committed it to see whether it works or not but I guess it won't work because I tried 444 before and it didn't work.

Reply:
Oops, it worked :sweat_smile:, appearently it failed because of the `.next` directory.

Reply:
I guess 004 is a bit weird (root owns it but can't read it?) let's go with 444.